lib/Backend/GlobOptFields.cpp

```diff
@@ -689,8 +689,7 @@ GlobOpt::PreparePrepassFieldHoisting(Loop * loop)
 
                 // Set object type live on prepass so we can track if it got killed in the loop. (see FinishOptHoistedPropOps)
                 JsTypeValueInfo* typeValueInfo = JsTypeValueInfo::New(this->alloc, nullptr, nullptr);
-                typeValueInfo->SetSymStore(typeSym);
-                typeValueInfo->SetIsShared();
```

Comment:
Oh, this is not intended, will add it back. Thanks for the catch.

---
In reply to: [68347957](https://github.com/Microsoft/ChakraCore/pull/1152#discussion_r68347957) [](ancestors = 68347957)